roles/aks-deployment/files/dev/digitransit-deployer-dev.yml

```diff
@@ -85,13 +85,21 @@ spec:
               topologyKey: kubernetes.io/hostname
       containers:
       - name: digitransit-deployer
-        image: hsldevcom/digitransit-deployer:kubernetes
+        image: hsldevcom/digitransit-deployer:kubernetes #TODO use latest later
         imagePullPolicy: Always
         securityContext:
           allowPrivilegeEscalation: false
         env:
         - name: DEBUG
           value: "*"
+        # TODO enable these envs
+        # - name: ENVIRONMENT_TYPE
```

Comment:
Why these are commented? They can be enabled already.

Reply:
They are used for slack and the logic for how it decides which slack channel to use is hardcoded and I don't want it yet to confuse things by posting to monitoring channels used by existing environments.